crates/ra_syntax/src/grammar.ron

```diff
@@ -582,6 +582,7 @@ Grammar(
             options: [
                 ["pat", "Pat"],
                 ["initializer", "Expr"],
+                ["initializer_macro", "MacroCall"],
```

Comment:
Why do we need this accessor? macro should be treated as an expression in this position. Perhaps there's a bug in the parser?

Reply:
macro is not an expr😖